crates/uv/tests/it/lock_conflict.rs

```diff
@@ -546,16 +546,16 @@ fn extra_multiple_not_conflicting2() -> Result<()> {
         project4 = ["sortedcontainers==2.4.0"]
         "#,
     )?;
-    uv_snapshot!(context.filters(), context.lock(), @r###"
+    uv_snapshot!(context.filters(), context.lock(), @r"
     success: false
     exit_code: 1
     ----- stdout -----
 
     ----- stderr -----
-      × No solution found when resolving dependencies:
+      × No solution found when resolving dependencies for split (included: project[extra2], project[project3]; excluded: project[extra1], project[project4]):
```

Comment:
This sort of seems like noise to a user?

Reply:
I don't think this context makes the following error message any easier to understand? I think it makes it more confusing, even. I think showing the splits for the markers are confusing too though... so I guess we can track improving this as a larger goal.